content/docs/configuration/marznode.md

```diff
@@ -1,4 +1,93 @@
 ---
-title: Marznode
+
+title: Marznode  
 weight: 2
+
+---
+
+### Easy Installation Method:
+
+1. Installing Docker and pulling Marznode with this command:
+
+```bash
+apt-get update -y && apt-get upgrade -y && curl -fsSL https://get.docker.com | sh && git clone https://github.com/khodedawsh/marznode && cd marznode && docker compose up -d
```

Comment:
why not marzneshin? 
`clone https://github.com/khodedawsh/marznode`

Reply:
done